test/acceptor_SUITE.erl

```diff
@@ -1373,3 +1434,12 @@ do_os_supports_reuseport() ->
 		{{unix, linux}, {3, Minor, _}} when Minor >= 9 -> true;
 		_ -> false
 	end.
+
+do_os_supports_local_sockets() ->
+	case os:type() of
+		{unix, _} -> true;
+		_ -> false
+	end.
+
+do_tempname() ->
+	lists:reverse(tl(lists:reverse(os:cmd("mktemp -u")))).
```

Comment:
There's actually better, `lists:droplast(os:cmd(...))`. I'll do it when merging.